<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDSURocketProject</class>
 <widget class="QMainWindow" name="SDSURocketProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RocketProject</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="Readout1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Readout0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="MPV_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MPV's</string>
    </property>
   </widget>
   <widget class="QLabel" name="ignite_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Igniter</string>
    </property>
   </widget>
   <widget class="QLabel" name="pstate_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pressurize</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="MPV_Safety">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>41</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="MPV_Key">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>41</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Ign_Key">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>41</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Ign_Safety">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>41</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Pressure_Key">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Readout2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>165</y>
      <width>631</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4500</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pstate_label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>125</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HE BOTTLE</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>245</y>
      <width>631</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4500</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pstate_label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>205</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HE REG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>145</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 PSI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>707</x>
      <y>145</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4500 PSI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>225</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 PSI</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>325</y>
      <width>631</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pstate_label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PNU</string>
    </property>
   </widget>
   <widget class="QLabel" name="pstate_label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Extra</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>405</y>
      <width>631</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Readout3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>305</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 PSI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>385</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 PSI</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide Extra</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Alert1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>450</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DISCONNECTED</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>707</x>
      <y>225</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4500 PSI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>714</x>
      <y>305</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>150 PSI</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ign_state">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ch4_state">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="lox_state">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>105</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TReadout">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="pstate_label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temp</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TReadout_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="pstate_label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tank P</string>
    </property>
   </widget>
   <zorder>line_3</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>Readout1</zorder>
   <zorder>Readout0</zorder>
   <zorder>MPV_label</zorder>
   <zorder>ignite_label</zorder>
   <zorder>pstate_label</zorder>
   <zorder>MPV_Safety</zorder>
   <zorder>MPV_Key</zorder>
   <zorder>Ign_Key</zorder>
   <zorder>Ign_Safety</zorder>
   <zorder>Pressure_Key</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>Readout2</zorder>
   <zorder>progressBar0</zorder>
   <zorder>pstate_label_2</zorder>
   <zorder>progressBar1</zorder>
   <zorder>pstate_label_3</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>progressBar2</zorder>
   <zorder>pstate_label_4</zorder>
   <zorder>pstate_label_5</zorder>
   <zorder>progressBar3</zorder>
   <zorder>Readout3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>checkBox</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>ign_state</zorder>
   <zorder>ch4_state</zorder>
   <zorder>lox_state</zorder>
   <zorder>TReadout</zorder>
   <zorder>pstate_label_6</zorder>
   <zorder>TReadout_2</zorder>
   <zorder>pstate_label_7</zorder>
   <zorder>Alert1</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
